ble_controller/CHANGELOG.rst

```diff
@@ -5,6 +5,19 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+ble_controller 0.3.0-3.prealpha
+*******************************
+
+Changes
+=======
+* nRF52833 is now supported.
```

Comment:
I would remove this entry. You have a different commit that fixes the issue with the build files. And since the support wasn't added in this release it seems wierd to mention it here. And the "Added" segment for the 0.3.0-2 release should cover it.

Reply:
Discussed offline. Should have been "Added" instead of "Changes". Now it is more clear